docs/security.md

```diff
@@ -11,11 +11,51 @@ Before creating / modifying any GitHub Actions workflow make sure you're familia
 In this page we are also providing guidance on how to setup common tooling related to security.
 
 - [Security best practices](#security-best-practices)
+  - [SHA Pinning for GitHub Actions](#sha-pinning-for-github-actions)
   - [Secrets detection](#secrets-detection)
     - [First setup](#first-setup)
     - [Updating new/old secrets to the baseline](#updating-newold-secrets-to-the-baseline)
       - [Excluding multiple secrets via regex](#excluding-multiple-secrets-via-regex)
 
+## SHA Pinning for GitHub Actions
+
+To enhance supply chain security, it's crucial to pin GitHub Actions to specific SHA hashes instead of using mutable tag references. This prevents potential supply chain attacks where malicious code could be injected by compromising action repositories.
+
+### Why SHA Pinning Matters
```

Comment:
again no duplicated docs here, I think here we just need to have a link to the action (this section was more to educate about the security approach rather than documenting security related actions)